tests/dialects/test_bigquery.py

```diff
@@ -2877,6 +2877,12 @@ def test_json_lax(self):
         self.validate_identity("LAX_INT64(PARSE_JSON('10'))")
         self.validate_identity("""LAX_STRING(PARSE_JSON('"str"'))""")
 
+    def test_utc_time(self):
+        self.validate_identity("UTC_TIME()").assert_is(exp.UtcTime)
+        self.validate_identity("UTC_TIME(6)").assert_is(exp.UtcTime)
+        self.validate_identity("UTC_TIMESTAMP()").assert_is(exp.UtcTimestamp)
+        self.validate_identity("UTC_TIMESTAMP(6)").assert_is(exp.UtcTimestamp)
```

Comment:
These are testing non-existent BigQuery functions. We should always test against the engine when transpiling.